src/containers/org/members/RolesSection.js

```diff
@@ -2,12 +2,15 @@
  * @flow
  */
 
-import React, { useState } from 'react';
+import React, { useCallback, useState } from 'react';
 
 import styled from 'styled-components';
-import { Button, Checkbox } from 'lattice-ui-kit';
+import { Button, Checkbox, SearchInput } from 'lattice-ui-kit';
 import { ValidationUtils } from 'lattice-utils';
-import type { Organization, Role, UUID } from 'lattice';
+import { debounce } from 'lodash';
```

Comment:
There was something about lodash and webpack tree shaking... please import as follows:

```
import _debounce from 'lodash/debounce';
```